assets/runtime/functions

```diff
@@ -1448,12 +1448,30 @@ generate_ssh_host_keys() {
   chmod 0644 ${GITLAB_DATA_DIR}/ssh/*.pub
 }
 
+configure_container_timezone() {
+  TIMEZONE=${TZ:-UTC}
```

Comment:
This default config I think should be set in `env-defaults` if it is in fact required.

Reply:
+1

Reply:
PR Updated;